assignment-2-ruby/ruiqu/customized_room_cleaner.rb

```diff
@@ -0,0 +1,118 @@
+# Customize the room_cleaner program.
+# Possibilities:
+
+# Accept user input to get room names (or offer a selection of available rooms)
+# Create an invoice-style output at the end of the program that varies based on the rooms and processes completed.
+
+
+def dust(room)
+  puts "The #{room} has been dusted"
+end
+
+def vacuum(room)
+  puts "The #{room} has been vacuumed."
+end
+
+def make_bed
+  puts "The bed has been made."
+end
+
+def mop_floor
+  puts "The floor has been mopped."
+end
+
+def wash_dishes
+  puts "The dishes have been washed."
+end
+
+def clean_toilet
+  puts "The toilet has been cleaned."
+end
+
+def clean_room(room)
+  if room == "bedroom"
+    vacuum(room)
+    make_bed
+  elsif room == "kitchen"
+    mop_floor
+    wash_dishes
+  elsif room == "bathroom"
+    mop_floor
+    clean_toilet
+  else
+    puts "The #{room} requires no specific cleaning processes."
+  end
+
+  mop_floor if room == "kitchen" || room == "bathroom"
```

Comment:
I think you don't need this line as it's covered by the if/else block above?

Reply:
right, removed!